src/test/java/ua/ithillel/lesson14/ThreadSafeListTest.java

```diff
@@ -0,0 +1,57 @@
+package ua.ithillel.lesson14;
+
+import org.junit.jupiter.api.Test;
+import ua.ithillel.lesson14.item3.ThreadSafeList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ThreadSafeListTest {
+    List<String> list = new ArrayList<>();
```

Comment:
і що ми тут тестуємо, ArrayList? 
```suggestion
    ThreadSafeList list = new ThreadSafeList<>();
```